WalletWasabi.Fluent/ViewModels/Wallets/WalletRenameViewModel.cs

```diff
@@ -0,0 +1,45 @@
+using System.Reactive;
+using ReactiveUI;
+using WalletWasabi.Fluent.Models.Wallets;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets;
+
+[NavigationMetaData(Title = "Rename Wallet")]
+public partial class WalletRenameViewModel : DialogViewModelBase<Unit>
+{
+	[AutoNotify] private string _newWalletName;
+
+	private WalletRenameViewModel(IWalletModel wallet)
+	{
+		_newWalletName = wallet.Name;
+
+		this.ValidateProperty(
+			x => x.NewWalletName,
+			errors =>
+			{
+				if (UiContext.WalletRepository.ValidateWalletName(NewWalletName) is { } error)
```

Comment:
https://github.com/zkSNACKs/WalletWasabi/pull/11652/files#r1387779289

NewWalletName == CurrentWalletName, There shouldn't be an error

![image](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/1fe85b5c-ad9d-4ff1-adb9-21203b8305bf)

Reply:
Fixed